### ISSUE TITLE
Display tunga history in developer profile

### ISSUE THREAD

**Issue by reiniervs** [tunga-io/tunga-web#400]
## I'm submitting a ...

<pre><code>
[ ] Bug report 
[x] Feature request
[ ] Regression (behavior that used to work and stopped working in a new release)
[ ] Enhancement
</code></pre>

## Current behavior

At the moment we can't see on the platform which developers are already busy on a project or how many projects they've done for us

## Expected behavior

I would like to see in the Tunga profile of the developers whether they are already working on a project (could be automated when they are assigned to a project in the Tunga platform?) and how many projects they've done for us in the past.

## What is the motivation / use case for changing the behavior?

Of the 250+ developers, 1/5th are currently working on projects and they are mostly the ones who get new projects once we're done, since the PM's already know them and trust them. To scale however, we have to rotate projects on more developers. Indicating who is already busy doing what and who has been engaged in the platform for what time is an incentive for PM's to find other developers.

## Requested Delivery Date (feature requests only):
01/02/2019

## Please tell us about your environment (optional for feature requests)

<pre><code>

Browser:
- [ ] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
- [ ] Other version XX 

Operating System:
- [ ] macOS version XX
- [ ] Windows version XX
- [ ] Linux  XX
- [ ] Android version XX
- [ ] iOS version XX
- [ ] Windows Phone version XX
- [ ] Other version XX 

Environment/ Server:
- [ ] Production
- [ ] Staging
- [ ] Test
- [ ] Development

For Developer issues:
- Node version: XX 
- Platform: 

Others:

Requested Delivery Date: dd/MMM/YYYY

</code></pre>

**Comment by davidsemakula** [comment]
@bartleijssenaar Assigning to your first for design, feel free to assign back to me when that is completed.

**Comment by bartleijssenaar** [comment]
Design done

Results:
Project shows up on the experience list of the dev
Badge

**Comment by gerodur** [comment]
@bartleijssenaar please share the design for this

**Comment by bartleijssenaar** [comment]
We don't need a separate design for this, this will be in the same format for an external project or work experience. 

![Screenshot 2019-06-14 at 16 36 22](https://user-images.githubusercontent.com/12690878/59516897-9c625380-8ec2-11e9-8481-824325efb43e.png)

We will add behind the project or job title in red (hex #DA3451) for Tunga.

The data for this needs to come directly from our system.

**Comment by gerodur** [comment]
@bartleijssenaar  according to the story details, we also need a field where users can see the number of projects (as an integer) a developer has worked on for Tunga. Where do you think we can put this?

**Comment by gerodur** [comment]
Also, if we are going to have only one design, it appears we would also need an "Active" tag on Tunga projects that are currently going on, so everyone can see the developer is occupied @bartleijssenaar. What do you think

**Comment by bartleijssenaar** [comment]
I think the automated approach would be better, I think it is wise that they can fill in the details of their responsibilities in their profile tho. An active tag would be great as well. This can be just a green dots behind the project title. wdyt?

**Comment by gerodur** [comment]
Sounds good! 3 green dots below the Project title, left aligned for active projects. Do you think that could work @bartleijssenaar

**Comment by bartleijssenaar** [comment]
Why 3? Is one also ok? :-)

**Comment by gerodur** [comment]
Yup. One works. Though if it is one, I suggest it be either on the left or right of the project title to save space and increase context @bartleijssenaar

**Comment by bartleijssenaar** [comment]
Both options work for me. Your call.

**Comment by gerodur** [comment]
Left of Project title it is
